### PR TITLE
Remove Scala ServiceMonitor

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

File: service-monitor/pom.xml [hunk 1]
```diff
@@ -16,20 +16,6 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.scala-lang</groupId>
-            <artifactId>scala-library</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.reactivex</groupId>
```
Comment (bjorncs) [inline service-monitor/pom.xml]: Remove all rxjava dependencies from parent pom.xml

File: service-monitor/pom.xml [hunk 2]
```diff
@@ -72,17 +58,6 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.json4s</groupId>
-            <artifactId>json4s-native_${scala.major-version}</artifactId>
```
Comment (bjorncs) [inline service-monitor/pom.xml]: Remove json4s-native from parent pom.xml

**Review by bjorncs — CHANGES_REQUESTED** [review]

File: service-monitor/pom.xml (hunk 1 above)
Comment (hakonhall) [reply service-monitor/pom.xml]: I failed to find any dependencies in parent pom.xml that had to be removed from these changes, except the one you commented on below. Is there any particular dependency you're referring to?

File: service-monitor/pom.xml (hunk 2 above)
Comment (hakonhall) [reply service-monitor/pom.xml]: Done

**Comment by hakonhall** [comment]
@bjorncs Please take another look

**Review by bjorncs — APPROVED** [review]

**Pull request merged**